### PR TITLE
CF List setting (FLOWS)

### PR DESCRIPTION
**Pull request by ambuznego**
- Per discussion w/ team, the overwriting selection will be `disabled`. Since the default setting is `enabled`, if a device has a label that has CF List `disabled` but the device itself (or another label) kept the default `disabled` setting, `disabled` is the setting that is sent to router.

### CODE REVIEW

File: assets/js/components/flows/nodes/DeviceNode.jsx [hunk 1]
```diff
@@ -7,8 +7,10 @@ import AdrTag from "../../../../img/adr/adr-node-tag.svg";
 import MultiBuyTag from "../../../../img/multi_buy/multi-buy-node-tag.svg";
 import SelectedNodeIcon from "./SelectedNodeIcon";
 import AlertTag from "../../../../img/alerts/alert-node-tag.svg";
+import { SettingOutlined } from "@ant-design/icons";
 
 export default ({ data, fromSidebar, selected }) => {
+  console.log(data);
```
Comment (vicmgs) [inline assets/js/components/flows/nodes/DeviceNode.jsx:13]: log

File: assets/js/components/flows/nodes/DeviceNode.jsx [hunk 2]
```diff
@@ -7,8 +7,10 @@ import AdrTag from "../../../../img/adr/adr-node-tag.svg";
 import MultiBuyTag from "../../../../img/multi_buy/multi-buy-node-tag.svg";
 import SelectedNodeIcon from "./SelectedNodeIcon";
 import AlertTag from "../../../../img/alerts/alert-node-tag.svg";
+import { SettingOutlined } from "@ant-design/icons";
```
Comment (vicmgs) [inline assets/js/components/flows/nodes/DeviceNode.jsx:10]: not used

Comment (ambuznego) [reply assets/js/components/flows/nodes/DeviceNode.jsx]: guilty as charged

File: assets/js/components/flows/FlowsWorkspace.jsx [hunk 3]
```diff
@@ -160,6 +160,17 @@ export default ({
     );
   };
 
+  const onCFListUpdate = (id, cfListEnabled) => {
+    console.log(cfListEnabled);
```
Comment (vicmgs) [inline assets/js/components/flows/FlowsWorkspace.jsx:164]: log

Comment (ambuznego) [reply assets/js/components/flows/FlowsWorkspace.jsx]: fixed

File: assets/js/components/flows/nodes/DeviceNode.jsx (hunk 1 above)
Comment (ambuznego) [reply assets/js/components/flows/nodes/DeviceNode.jsx]: fixed

**Review by vicmgs — APPROVED** [review]
perfecto

**Pull request merged**